lib/src/outputs/async_output.dart

```diff
@@ -0,0 +1,141 @@
+import 'dart:collection';
```

Comment:
Since Dart doesn't have `yeld` and `sleep`, any code in the thread will run straightforward (specially in the browser). The concept to release CPU of the current thread doesn't exists for Dart. If you use a Stream, doesn't mean that it will `yeld` and release CPU for the main software, or even parallelize anything .

This is why `AsyncOutput ` has `_maximumFlushTime`. If method `flush` takes more than `_maximumFlushTime` ms, it stops flushing and schedules a call to flush some ms later, mimicking a `yeld` behaviour of the thread.

Reply:
Dart does have a `yield` keyword. With which you can give control back and the scheduler can schedule a different micro task (or not).

I think it would be a good idea to deal with this contribution in a different pull request. Can you open a new PR for this particular output?